interface-definitions/interfaces-virtual-ethernet.xml.in

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="interfaces">
+    <children>
+      <tagNode name="virtual-ethernet" owner="${vyos_conf_scripts_dir}/interfaces-virtual-ethernet.py">
+        <properties>
+          <help>Virtual Ethernet Interface (veth)</help>
+          <priority>300</priority>
+          <constraint>
+            <regex>veth[0-9]+</regex>
+          </constraint>
+          <constraintErrorMessage>Virutal Ethernet interface must be named vethN</constraintErrorMessage>
+          <valueHelp>
+            <format>vethN</format>
+            <description>Virtual Ethernet interface name</description>
+          </valueHelp>
+        </properties>
+        <children>
+          #include <include/interface/address-ipv4-ipv6-dhcp.xml.i>
+          #include <include/interface/description.xml.i>
+          #include <include/interface/disable.xml.i>
+          #include <include/interface/vrf.xml.i>
+          <node name="peer">
+            <properties>
+              <help>Peer options</help>
+            </properties>
+            <children>
+              <leafNode name="name">
```

Comment:
Maybe rename the entire node to `peer-name`?

Reply:
I use node `peer` and leaf `name` as it seems the following step/request will be add the ability to set address to the peer site
something like:
```
set interfaces virtual-ethernet veth1010 address '10.0.0.10/24'
set interfaces virtual-ethernet veth1010 peer name 'veth0'
set interfaces virtual-ethernet veth1010 peer address 10.0.0.254/24
```

Reply:
Done!
```
set interfaces virtual-ethernet veth0 peer-name 'veth1010'
set interfaces virtual-ethernet veth1010 peer-name 'veth0'

```